jprm/__init__.py

```diff
@@ -348,9 +350,9 @@ def build_plugin(path, output=None, build_cfg=None, version=None, dotnet_config=
         logger.error(stderr)
         exit(1)
 
-    build_command = "dotnet publish --nologo" \
+    build_command = "dotnet publish --nologo --no-restore" \
         " --configuration={dotnet_config} --framework={dotnet_framework}" \
-        " --output={output} /p:Version={version}"
+        " --output={output} -p:Version={version}"
```

Comment:
Have you tested this change?
If there's a `/` instead of a `-` for a cli switch in my code, there's probably a reason for it, can't remember if this one was important tho.